src/app/modules/time-entries/pages/time-entries.component.spec.ts

```diff
@@ -381,7 +381,7 @@ describe('TimeEntriesComponent', () => {
       uri: 'http://testing.is.fun',
       activity_id: 'sss',
       project_id: 'id',
-      start_date: new Date(new Date().setHours(0, 0, 0, 0)),
+      start_date: new Date(),
```

Comment:
Is necessary specify the format to start_hour,
Example to get the current hour:
`new Date().getHours()`